jetstream/statistics.py

```diff
@@ -392,6 +404,48 @@ def transform(
         )
 
 
+@attr.s(auto_attribs=True)
+class PerClientDAUImpact(BootstrapMean):
+    def transform(
+        self,
+        df: DataFrame,
+        metric: str,
+        reference_branch: str,
+        experiment: Experiment,
+        analysis_basis: parser_metric.AnalysisBasis,
+        segment: str,
+        total_enrolled_clients: int,
+    ) -> StatisticResultCollection:
+        bootstrap_results = super().transform(
+            df,
+            metric,
+            reference_branch,
+            experiment,
+            analysis_basis,
+            segment,
+            total_enrolled_clients,
+        )
+        enrollment_fraction = experiment.bucket_config.count - experiment.bucket_config.start
+        reachable_population = total_enrolled_clients / enrollment_fraction
+
+        results = []
+        for branch in experiment.branches:
+            branch_data = [
+                x
+                for x in bootstrap_results.data
+                if x.branch == branch.slug and x.comparison == "difference"
+            ]
+            for d in branch_data:
+                d.point = d.point * reachable_population
```

Comment:
let's multiply here by `total_enrolled_clients` instead. The `reachable_population` is a mess because that depends so strongly on the targeting. Let's just use the actual experiment population instead.